<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finderdialog</class>
 <widget class="QDialog" name="finderdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="caseCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>128</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Точный поиск</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findNextButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAllButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>103</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Заменить все</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Заменить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>127</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="findLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="replaceLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
